app/helpers/likes_helper.rb

```diff
@@ -0,0 +1,2 @@
+module LikesHelper
```

Comment:
- [ ] You were requested not to use file generation command like scaffold command, but you used it anyways. Kindly remove this scaffold file.